test/e2e/prune_test.go

```diff
@@ -280,6 +280,23 @@ var _ = Describe("Podman prune", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToStringArray()).To(HaveLen(0))
+
+		// Create new network.
+		session = podmanTest.Podman([]string{"network", "create", "test1", "--label", "foo"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		// Remove all unused networks.
+		session = podmanTest.Podman([]string{"system", "prune", "-f", "--filter", "label!=foo"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
```

Comment:
I think we should check the output here as well.  At the moment, we only know that it succeeded.

Reply:
done